src/YieldDisburser.sol

```diff
@@ -18,7 +18,8 @@ contract YieldDisburser is OwnableUpgradeable {
     uint48 public minimumTimeBetweenClaims;
     mapping(address => uint256[]) public holderToDistribution;
 
-    event BaseYieldDistributed(uint256 amount, address project);
+    event ProjectDistributionStats(uint256 votedYield, uint256 baseYield, uint256 percentage, address project);
+    event TotalVotesForProject(uint256 totalVotes, address project);
```

Comment:
`address project` field should be "indexed" so it's easier to search for later on

Reply:
resolved with 5584d2f